docs/_css/term.css

```diff
@@ -0,0 +1,15 @@
+pre.codehilite > code,
+div.codehilite > pre {
+  background-color: black;
```

Comment:
The code blocks seem to not have very much contrast on a black background. Pygments seems to work better on a light background. Maybe #F0F8FF (aka AliceBlue 😜) might be a good fit?